config/locales/en.yml

```diff
@@ -899,7 +899,6 @@ en:
   bold_label_colon_value_html: "<strong>%{label}:</strong> %{value}"
   # Text inserted into a textarea when the bold button is clicked but no text was selected
   bold_text: bold text
-  bounding_box: Bounding Box
```

Comment:
Let me know if I should remove this from all locale files (or if there's an automated way to do it).

Reply:
en.yml is the only source file, so the next time we import new translations from Crowdin it will automatically get removed from all the other locale files.